src/workspace-mapping.ts

```diff
@@ -186,3 +188,50 @@ export function handleWorkspaceMembersUpdated(event: WorkspaceMembersUpdated): v
 
 	entity.save()
 }
+
+export function handleDisburseReward(event: DisburseReward): void {
```

Comment:
can we just use a common function for handling disburseReward from grant & workspace?

Reply:
Updated. Can you check?